### DISCUSSION TITLE
Defining TLS certificates and server config (inc. port, https mode) in a config file rather than CLI

### DISCUSSION THREAD

**Discussion by noahnu** [storybookjs/storybook#21383] [Ideas]
It'd be nice to support configuring the dev server from a configuration file rather than from the CLI. I believe this was possible in older versions of Storybook (not 100% sure).

**Comment by JoJk0** [comment]
I have a setup script that points to correct certs and I would like to use it to setup Storybook server rather than having multiple sources of truth.
